pkg/json/json_test.go

```diff
@@ -7,7 +7,7 @@ import (
 	"net/http/httptest"
 	"testing"
 
-	"github.com/vladyslavpavlenko/genesis-api-project/internal/utils"
+	json2 "github.com/vladyslavpavlenko/genesis-api-project/pkg/json"
```

Comment:
better give normal name like jsonhelper

Reply:
oops, haven’t noticed this one 😬 thanks!